packages/plugin-dom-layout/src/DomReconciliationEngine.ts

```diff
@@ -151,10 +157,16 @@ export class DomReconciliationEngine {
                         const ancestorObjectId = this._fromItem.get(ancestorWithRendering);
                         if (ancestorObjectId && !this._diff[ancestorObjectId]) {
                             const parentObject = this._objects[ancestorObjectId];
-                            const nodes = this._items.get(parentObject.object);
-                            mapOldIds.set(parentObject.object, new Set([ancestorObjectId]));
-                            this._rendererTreated.delete(parentObject.object);
-                            this._diffObject(renderings, parentObject.object, nodes, mapOldIds);
+                            if (!parentObject) {
+                                console.warn(
+                                    'This parent are removed previously, at least one reference was wrong.',
```

Comment:
idem avec le parent

Reply:
Trying to remove a parent that has already been removed. One of the references is wrong.